.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,51 @@
+name: Integration Tests
```

Comment:
`m`: As this is currently only specific to CocoaPods, I think we should rename this to CocoaPods Tests including the file.